cobbler/modules/managers/in_tftpd.py

```diff
@@ -150,6 +150,33 @@ def add_single_distro(self, distro):
         self.tftpgen.copy_single_distro_files(distro, self.bootloc, False)
         self.write_boot_files_distro(distro)
 
+    def sync_systems(self, systems, verbose: bool = True):
+        """
+        Write out specified systems files to /tftpdboot
```

Comment:
I would have written this as
"Write out specified systems as separate files to /tftpdboot"